### DISCUSSION TITLE
Not able to create Order using Retail APIs with Managed Identity

### DISCUSSION THREAD

**Discussion by nikithaatla** [microsoft/Dynamics365Commerce.Solutions#40] [Q&A]
![image](https://user-images.githubusercontent.com/88876701/229003558-24d3c200-9669-48b7-a2a0-ac36f9cfbdd5.png)

**Comment by Pikhulya** [comment]
Please clarify what the calling application, you are considering to use Managed Identity while communicating to CSU (Retail Server), is.

P. S.
A suggestion: try to avoid asking multiple different questions in the same formal question as it is difficult to maintain and track them.

**Reply by nikithaatla** [reply]
Azure Web APP is calling the Retail Server APIs using Managed Identity.
We are successfully able to get the token from the azure, but Create cart API is failing with above mentioned error.
Note: Able to retrieve products from SearchByCategory without any issues.
